Update active-directory-configurable-token-lifetimes.md

chadhasbrook opened the pull request:
Added an additional "Note" Section that describes SharePoint Online is not supported for Token Lifetime policies.

I've tested the SPO and confirmed with other GTP Team members from CXP Identity engineering that this is not a supported scenario.  We should call out the SPO information as customers are finding it difficult linking the documentation.

azurecla commented:
Hi __@pehteh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (pehteh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jeffgoll commented:
#sign-off

azurecla commented:
Hi __@pehteh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (pehteh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
#sign-off

chadhasbrook committed:
Update active-directory-configurable-token-lifetimes.md

Added an additional "Note" Section that describes SharePoint Online is not supported for Token Lifetime policies.
